src/apps/console/routes/_main+/$account+/_layout.tsx

```diff
@@ -222,7 +199,7 @@ const DevicesMenu = ({ devices }: { devices: IConsoleDevicesForUser }) => {
     }
   };
   return (
-    devices?.length > 0 && (
+    device && (
```

Comment:
if not, the device returns null. not need to && operator it may return false as it's not inside JSX.